omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -142,7 +151,8 @@ public String handleSubmission(WebRequest request, HttpSession httpSession, Mode
 	        @RequestParam(required = false, value = "confirm") String confirm,
 	        @RequestParam(required = false, value = "forcePassword") Boolean forcePassword,
 	        @RequestParam(required = false, value = "roleStrings") String[] roles,
-	        @RequestParam(required = false, value = "createNewPerson") String createNewPerson,
+	        @RequestParam(required = false, value = "createNewPerson") String createNewPerson, 
+	        @RequestParam(required = false, value = "providerCheckBox") String addToProviderTableOption,
```

Comment:
Naah. Not really. If the checbox is ticked, it stores the value attribute of the check box which is "addToProviderTable" else it is null